Samples/excel-keyboard-shortcuts/README.md

```diff
@@ -0,0 +1,114 @@
+
+## Summary
+
+This sample shows how to set up a basic Excel add-in project that utilizes keyboard shortcuts. Currently, the shortcuts are configured to show and hide the taskpane as well as cycle through colors for a selected cell. Keyboard shortcuts can be used to achieve any action within the add-in runtime.
+
+> **Note:** The features used in this sample are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you'll need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is to sign up for an Office 365 subscription. If you don't already have an Office 365 subscription, get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
+
+## Features
+
+- Add keyboard shortcuts to your Office Add-in. 
+- Enable users to use those keyboard shortcuts to invoke any action within the Office Add-in runtime.
+
+## Applies to
+
+-  Excel on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+To use this sample, you'll need to [join Office Insider](https://insider.office.com/join).
+
+Before running this sample, you need a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/en/) installed on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Office Add-in share global data with a shared runtime | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 11-5-2020 | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+
+## Scenario: Open/Close taskpane and modify cell color
+
+This sample adds three different shortcuts to the Office Add-in. This enables the user to:
+- Use the "Ctrl+Shift+Up" keyboard shortcut to open the taskpane.
+- Use the "Ctrl+Shift+Down" keyboard shortcut to close the taskpane.
+- Use the "Ctrl+Shift+O" keyboard shortcut to cycle through colors for a selected cell.
+
+## Build and run the solution
+
+1. In the command prompt, run the command `npm install`.
+2. Run the command `npm run start`. This starts the node server, and opens Excel on the desktop.
+
+If you're running Excel on the web or Mac, run `npm run start:web` instead and see the following articles for instructions on how to sideload:
+
+- [Sideload Office Add-ins in Office on the web for testing](https://docs.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing) 
+- [Sideload Office Add-ins on iPad and Mac for testing](https://docs.microsoft.com/office/dev/add-ins/testing/sideload-an-office-add-in-on-ipad-and-mac)
```

Comment:
I don't think this will work on iPad, will it? Should we add a note here just to call that out? I know the article shows sideload for Mac and iPad, but I think really we are just point them to the Mac instructions. Is this the case?